.github/workflows/pr-build.yaml

```diff
@@ -12,6 +12,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
+    env:
+      # Set LANG=C to simulate least-common-denominator target deployment environments:
+      LANG: C
```

Comment:
Tested this over in my fork at https://github.com/JoshRosen/sjsonnet/pull/1 and confirmed that it reproduced the bug.